parsers/imageAltTag.js

```diff
@@ -0,0 +1,47 @@
+var _ = require('lodash');
+var cheerio = require('cheerio');
+var moment = require('moment');
+var debug = require('debug')('parser:imageAltTag');
+var parse = require('./lib/parse');
+
+/*
+ * Extract alt text from img tags
+ */
+
+var PARSER_COMPLETED_SUCCESSFULLY = false;
+var FACEBOOK_ALT_TEXT_START = 'Image may contain:';
+
```

Comment:
Here you're using a text as reference point; Is an approach that might work, but has to be used least as possible, because in Facebook interfaces with different language, will not appear `Image may contain:` String.

For example, Portuguese has not such feature:
![image](https://cloud.githubusercontent.com/assets/89555/23725355/f5a574e8-042e-11e7-8201-0f0833b619b2.png)